### PR TITLE
Improve diamond shape dimensions

### PR DESCRIPTION
**Pull request by Fausto95**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -742,10 +742,10 @@ function getArrowPoints(element: ExcalidrawElement) {
 }
 
 function getDiamondPoints(element: ExcalidrawElement) {
-  const topX = Math.PI + element.width / 2;
-  const topY = element.height - element.height;
+  const topX = Math.floor(element.width / 2) + 1;
```
Comment (vjeux) [inline src/index.tsx]: Why the +1?

Comment (vjeux) [reply src/index.tsx]: And why the floor?

Comment (Fausto95) [reply src/index.tsx]: Rough does not accept these numbers to be 0, I add 1 to avoid errors

Comment (vjeux) [reply src/index.tsx]: Can you add a comment saying that? Otherwise it’s an odd decision

Comment (Fausto95) [reply src/index.tsx]: Sure!

Comment (vjeux) [reply src/index.tsx]: What is the error btw, that seems odd that rough would throw

Comment (Fausto95) [reply src/index.tsx]: `InvalidStateError: Failed to execute 'createPattern' on 'CanvasRenderingContext2D': The image argument is a canvas element with a width or height of 0.`

**Pull request merged**